contracts/universalSchemes/VestingScheme.sol

```diff
@@ -239,6 +240,7 @@ contract VestingScheme is UniversalScheme, ExecutableInterface {
             agreementsCounter++;
         // Log the new agreement:
             emit NewVestedAgreement(agreementsCounter-1);
```

Comment:
NewVestedAgreement can be remove here..no ?

Reply:
We can remove it only if you are OK with subtracting from the originally intended functionality of that event, which was to be thrown in both cases where an agreement is created.  Let me know if you're OK with that and I'll delete the line.

Reply:
please remove NewVestedAgreement here